client/a8c-for-agencies/hooks/use-onboarding-tours.ts

```diff
@@ -44,7 +47,17 @@ export default function useOnboardingTours() {
 		[ dispatch ]
 	);
 
-	return useMemo( () => {
+	const dismiss = useCallback( () => {
+		dispatch(
+			savePreference( A4A_ONBOARDING_TOURS_DISMISSED_PREFERENCE_NAME, { dismiss: false } )
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question:** Since I'll have to implement something similar soon, I thought I could get a clarification about it now: is it intentional to set `dismiss` to `false` when a user clicks the button?

I can see that later we're fetching the `A4A_ONBOARDING_TOURS_DISMISSED_PREFERENCE_NAME` preference value and assigning it to the `isDismissed` constant. I'm guessing that it'll return a falsy value (likely `null`) if the preference does not exist and a truthy value (an object) otherwise but I might be wrong. So does it even matter if the preference value is set to `{ dismiss: bool }` object or could it be a boolean from the very beginning?

Reply:
Thanks for catching this. I meant to use boolean here and forgot to change it. This was copiedfrom another line above.